Replace slashed zeros with dotted zeros.

Dedicated to eroen :-)


Author: wiktor w brodlo <wiktor@brodlo.net>
Modifies:
 U+0030 DIGIT ZERO
 U+2070 SUPERSCRIPT ZERO
 U+2080 SUBSCRIPT ZERO

--- a/leggie-12.bdf
+++ b/leggie-12.bdf
@@ -337,9 +337,9 @@ BITMAP
 00
 38
 44
-4C
 54
-64
+54
+54
 44
 38
 00
@@ -19638,9 +19638,9 @@ BBX 6 12 0 -2
 BITMAP
 38
 44
-4C
 54
-64
+54
+54
 44
 38
 00
@@ -19909,9 +19909,9 @@ BITMAP
 00
 38
 44
-4C
 54
-64
+54
+54
 44
 38
 ENDCHAR
--- a/leggie-18.bdf
+++ b/leggie-18.bdf
@@ -436,11 +436,11 @@
 3E00
 4100
 4100
-4300
-4500
+4100
 4900
-5100
-6100
+4900
+4900
+4100
 4100
 4100
 3E00
@@ -25832,11 +25832,11 @@
 3E00
 4100
 4100
-4300
-4500
+4100
+4900
+4900
 4900
-5100
-6100
+4100
 4100
 4100
 3E00
@@ -26188,11 +26188,11 @@
 3E00
 4100
 4100
-4300
-4500
+4100
 4900
-5100
-6100
+4900
+4900
+4100
 4100
 4100
 3E00
--- a/leggie-24.bdf
+++ b/leggie-24.bdf
@@ -535,12 +535,12 @@
 1F80
 30C0
 6060
-60E0
-6160
-6260
-6460
-6860
-7060
+6060
+6660
+6660
+6660
+6660
+6060
 6060
 30C0
 1F80
@@ -32025,12 +32025,12 @@
 1F80
 30C0
 6060
-60E0
-6160
-6260
-6460
-6860
-7060
+6060
+6660
+6660
+6660
+6660
+6060
 6060
 30C0
 1F80
@@ -32469,12 +32469,12 @@
 1F80
 30C0
 6060
-60E0
-6160
-6260
-6460
-6860
-7060
+6060
+6660
+6660
+6660
+6660
+6060
 6060
 30C0
 1F80
